packages/name-controller/src/NameController.ts

```diff
@@ -612,4 +614,34 @@ export class NameController extends BaseController<
       delete proposedNames[dormantSourceId];
     }
   }
+
+  #cleanupExpiredEntries(): void {
```

Comment:
Should we be super explicit and call this `removeExpiredEntries`?

Reply:
Renamed in https://github.com/MetaMask/core/pull/3748/commits/ae3c948627b10142bb87295a5c97e0e91bfa2794